TLM/TLM/LoadingExtension.cs

```diff
@@ -146,8 +146,7 @@ public class LoadingExtension : LoadingExtensionBase {
                 Log.Info("Removing Controls from UI.");
                 if (ModUI.Instance != null) {
```

Comment:
`ModUI.Instance` is derived from `MonoBehaviour`; this should be changed to `if (ModUI.Instance) ...`?

Reply:
Didn't make any changes here. It was not necessary for the task at hand (removal of Finalizers). So if it worked before, it should work afterwards.